config/config_det.template.yaml

```diff
@@ -61,25 +63,67 @@ make_detections.py:
 
 # Assess the final results
 merge_detections.py:
-  working_directory: ./output/det/CANTON/SWISSIMAGE_YEAR/
+  working_directory: output/det/CANTON/SWISSIMAGE_YEAR    # Must be the same dir as the OD for the automatic detection of  split_aoi_tils.geojson.
+  output_dir: ../modelMODEL/SWISSIMAGE_YEAR
   detections:
-    oth: oth_detections_at_0dot05_threshold.gpkg
+    oth: ../modelMODEL/SWISSIMAGE_YEAR/merged_detections_at_0dot05_threshold.gpkg
   distance: 10 # m, distance use as a buffer to merge close polygons (likely to belong to the same object) together
   iou_threshold: 0.1
   score_threshold: 0.05
   assess: 
     enable: False
     metrics_method: micro-average   # 1: macro-average ; 2: macro-weighted-average ; 3: micro-average
+
+# Merge years in one file or database
+compile_years.py:
+  canton: CANTON
+  working_directory: output/det/{canton}/modelMODEL
+  years: [YEAR1, YEAR2, YEAR3]          # choose years
```

Comment:
Change corresponding to the proposed edits in the bash files:
years: [YEARS_LIST]